Sources/sbg/main.swift

```diff
@@ -1,6 +1,7 @@
 import SBGCore
 
-let app = Application.default
+let appBuilder = ApplicationBuilder()
```

Comment:
I understand why you wanted to move whole initialization from `Application` class, but using `ApplicationBuilder` from other target is not the prettiest code in the world :) What do you think about leaving `default` value in `Application` class and use this `build` method there?

Reply:
What about this comment?